Plots/Colors/CB_Temperature.py

```diff
@@ -1,6 +1,6 @@
 """
 CB_Temperature.py
-=================
+================
```

Comment:
Sphinx will throw a warning with this saying that the title underline is too short. Make sure the underline is the same length as the title.